packages/react-scripts/config/webpack.config.js

```diff
@@ -184,14 +186,14 @@ module.exports = function(webpackEnv) {
       // There will be one main bundle, and one file per asynchronous chunk.
       // In development, it does not produce real files.
       filename: isEnvProduction
-        ? 'static/js/[name].[contenthash:8].js'
-        : isEnvDevelopment && 'static/js/bundle.js',
+        ? `static${commit}/js/[name].[contenthash:8].js`
```

Comment:
can we add a hyphen so static-<commit>?

Did you want to leave dev environments with this also? It means that every time you commit, the rebuild results would change, I think.

Reply:
:+1: 

I changed this to pass in a variable `BUILD_ID=foobar yarn build`, so the script will just use that.